modules/pipeline/providers/reconciler/task_reconciler.go

```diff
@@ -323,19 +322,6 @@ func (tr *defaultTaskReconciler) TeardownAfterReconcileDone(ctx context.Context,
 	if tr.edgeRegister.IsEdge() {
 		tr.edgeReporter.TriggerOnceTaskReport(task.ID)
 	}
-
-	// invalidate openapi oauth2 token
-	tokens := strutil.DedupSlice([]string{
```

Comment:
bootstrap token should be invalidated.
it's ttl is forever.

Reply:
and add todo

Reply:
done